src/app/admin/volunteers/shared/newvolunteer-register-form/newvolunteer-register-form.component.scss

```diff
@@ -0,0 +1,160 @@
+$gapBetweenFieldAndHint: "25px";
+$fontFamily: Roboto, sans-serif;
+
+$modalWindowWidth: 555px;
+$modalWindowHeight: 881px;
```

Comment:
No hardcoded widths and heights, UI should be responsive

Reply:
sorry - but it is properties from Figma 
![image](https://user-images.githubusercontent.com/4039092/103621633-2dce6600-4f3e-11eb-9b30-f79397867b30.png)

Reply:
changed to
$modalWindow_maxWidth: 555px;
$modalWindow_maxHeight: 881px;

![image](https://user-images.githubusercontent.com/4039092/103669683-562b8400-4f81-11eb-84b2-3913108a7b92.png)